package.json

```diff
@@ -89,6 +89,7 @@
     "rehype-react": "^7.0.3",
     "sass": "^1.44.0",
     "slugify": "^1.6.3",
+    "striptags": "^3.2.0",
```

Comment:
we had some nasty regex and I wanted to have a well maintained library for this (again and again surprisingly) complex topic of stripping html.